.env.example

```diff
@@ -0,0 +1,3 @@
+MONGODB_URI=mongodb://localhost:27017/db-name
```

Comment:
no need for this file. You don't need any environment variables to run this code